src/ape/_cli.py

```diff
@@ -1,66 +0,0 @@
-from typing import Dict
-
-try:
-    from importlib import metadata  # type: ignore
-except ImportError:
-    import importlib_metadata as metadata  # type: ignore
-
-import click
-import yaml
-
-from ape.plugins import clean_plugin_name
-
-
-def display_config(ctx, param, value):
-    # NOTE: This is necessary not to interrupt how version or help is intercepted
-    if not value or ctx.resilient_parsing:
-        return
-
-    from ape import project
-
-    click.echo("# Current configuration")
-    click.echo(yaml.dump(project.config.serialize()))
-
-    ctx.exit()  # NOTE: Must exit to bypass running ApeCLI
-
-
-class ApeCLI(click.MultiCommand):
-    _commands = None
-
-    @property
-    def commands(self) -> Dict:
-        if not self._commands:
-            entry_points = metadata.entry_points()
-
-            if "ape_cli_subcommands" not in entry_points:
-                raise Exception("Missing registered cli subcommands")
-
-            self._commands = {
-                clean_plugin_name(entry_point.name): entry_point.load
-                for entry_point in entry_points["ape_cli_subcommands"]
-            }
```

Comment:
So how would this change work with external plugins?

This code is how it's being handled now, via `entry_points`.

Reply:
For external plugins, they would install a separate `setup.py` which will it attach it to the `ape` CLI under the hood! So we wouldn't have to support this custom code for doing it

Reply:
I'm definitely looking forwards to this, this code isn't in the PR yet though right? I couldn't see it

Reply:
I can either do multiple small PRs in a staging branch or just keep pushing to this branch

Reply:
Seems like would be easier to work with another PR